authboss.go

```diff
@@ -39,6 +39,15 @@ func New() *Authboss {
 
 // Init authboss, modules, renderers
 func (a *Authboss) Init(modulesToLoad ...string) error {
+	// Create the hasher
+	// Have to do it in Init for backwards compatibility.
+	// If a user did not previously use defaults.SetCore() then they will
+	// suddenly start getting panics
+	// We also cannot use config.Defaults() so we can respect the user's BCryptCost
```

Comment:
Can't we just check if the BCryptCost is default and if so we can avoid the assignment?